DFC.ServiceTaxonomy.ApiFunction/OpenAPIDocs/GetAllOccupationsForSkillOpenApi.txt

```diff
@@ -0,0 +1,107 @@
+﻿openapi: 3.0.1
+info:
+  title: Get Occupations For Given Skill
+  description: 'Sample Description for Query'
+  version: 1.0.0
+servers:
+- url: __FunctionAppUrl__
+tags:
+- name: Get Occupations For Given Skill
+  description: Returns all occupation for a given skill
+paths:
+  /Execute/:
+    post:
+      tags:
+      - Skill
+      summary: Get occupations for a given skill.
+      description: Returns occupation for a given skill
+      operationId: GetOccupationsForSkill
+      requestBody:
+        description: enter esco skill uri to retrieve all Occupations for the Skill
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Skill'
+            example:   # Sample object
+                "skill": "http://data.europa.eu/esco/skill/68698869-c13c-4563-adc7-118b7644f45d"
+        required: true
+      responses:
```

Comment:
Do the documented responses match what the function actually returns?

For example, from reading the code (as opposed to running it), it looks like, if an occupation isn't found, the function will either return a 202 (No Content), or a 200 with no results. I can't see how a 404 would be returned.

The function can also return a 422 (Unprocessable Entity), but that status code isn't listed here.

Regarding the description of the 400 response, does the 'invalid id' refer to the passed in occupation id? If so, wouldn't the function follow the same path as 'occupation not found' and return a 200/202?

Looking at the paths/scenarios that do return a 400 from the function, there's either an issue with the request body or missing parameters, so we could probably have a more appropriate description to match those.

The function also returns a 400 when the function name is missing from the settings. We probably shouldn't return a client error for this, as it's really a server error (500 range).